modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheLifecycleAwareSelfTest.java

```diff
@@ -203,6 +184,13 @@ public TestEvictionPolicy() {
         @Override public boolean evictAllowed(Cache.Entry entry) {
             return false;
         }
+
+        /** {@inheritDoc} */
+        @Override public void start() {
```

Comment:
AFAIK, 
this method works incorrectly - please add assertion `assertNotNull(expCacheName)`.

we don't need this method - we just need to add `Serializable` marker to `TestLifecycleAware`.

Reply:
Eviction filter Serializable, already.